low-level/message_handlers/node_data_msg_handler.py

```diff
@@ -195,6 +195,59 @@ def initialize(self, conf_reader, msgQlist, product):
 
         self._import_products(product)
         cache_dir_path = os.path.join(DATA_PATH, self.CACHE_DIR_NAME)
+
+        # Persistent Cache for CPU sensor
+        self.CPU_SENSOR_DATA_PATH = os.path.join(cache_dir_path, f'CPU_SENSOR_DATA_{self.node_id}')
+
+        self.persistent_cpu_sensor_data = {}
+        if os.path.isfile(self.CPU_SENSOR_DATA_PATH):
+            self.persistent_cpu_sensor_data = store.get(self.CPU_SENSOR_DATA_PATH)
+        if self.persistent_cpu_sensor_data:
+            if self.persistent_cpu_sensor_data['cpu_fault'] == "True":
+                self.cpu_fault = True
+            else:
+                self.cpu_fault = False
+        else:
+            self.persistent_cpu_sensor_data = {
+                'cpu_fault' : str(self.cpu_fault),
+            }
+            store.put(self.persistent_cpu_sensor_data, self.CPU_SENSOR_DATA_PATH)
+
+        # Persistent Cache for Disk sensor
+        self.DISK_SENSOR_DATA_PATH = os.path.join(cache_dir_path, f'DISK_SENSOR_DATA_{self.node_id}')
+
+        self.persistent_disk_sensor_data = {}
+        if os.path.isfile(self.DISK_SENSOR_DATA_PATH):
+            self.persistent_disk_sensor_data = store.get(self.DISK_SENSOR_DATA_PATH)
+        if self.persistent_disk_sensor_data:
+            if self.persistent_disk_sensor_data['disk_fault'] == "True":
+                self.disk_fault = True
+            else:
+                self.disk_fault = False
+        else:
+            self.persistent_disk_sensor_data = {
+                'disk_fault' : str(self.disk_fault),

```

Comment:
What is 'self.disk_fault' before this, NULL? Why not set to false here?

Reply:
Already initialized to false along with the class member.